src/components/molecules/Visualizer/Plugin/types.ts

```diff
@@ -25,6 +25,7 @@ export type Reearth = {
   readonly widget?: Widget;
   readonly block?: Block;
   readonly scene: Scene;
+  readonly viewport?: Viewport;
```

Comment:
Is it possible to make viewport always available?

Reply:
Updated. Now is always avaliable.